docs/_docs/installation/kubernetes/generic-configuration.adoc

```diff
@@ -159,34 +163,35 @@ include::{script}[tags=create-configmap]
 
 === Creating Pod Configuration
 
-Now we will create a configuration for pods.
-In the case of stateless deployment, we will use a link:https://kubernetes.io/docs/concepts/workloads/controllers/deployment/[Deployment,window=_blank].
-For a stateful deployment, we will use a link:https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet,window=_blank].
+Now we create a configuration for pods that use a link:https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet,window=_blank].
 
 
 [tabs]
 --
 tab:Configuration without persistence[]
 
-Our Deployment configuration will deploy a ReplicaSet with two pods running Ignite {version}.
+Our non-persistent configuration deploys a ReplicaSet with two pods running GridGain Community Edition v. {version}.
+If you want to use Enterprise or Ultimate Edition, change the `spec.template.spec.containers.image` property in the configuration file.
 
-In the container's configuration, we will:
 
-* Enable the “ignite-kubernetes” and “ignite-rest-http” link:installation/installing-using-docker#enabling-modules[modules].
+In the container's configuration, we:
+
+* Enable the “ignite-kubernetes” and “ignite-rest-http” link:installation-guide/installing-using-docker#enabling-modules[modules].
 * Use the configuration file from the ConfigMap we created earlier.
 * Open a number of ports:
 ** 47100 — the communication port
 ** 47500 ­—­ the discovery port
 ** 49112 — the default JMX port
 ** 10800 — thin client/JDBC/ODBC port
 ** 8080 — REST API port
+* Create a volume for storing metadata.
 
-The deployment configuration file might look like as follows:
+The configuration file might look like as follows:
 
 .deployment.yaml
 [source, yaml,subs="attributes,specialchars"]
 ----
-include::{configDir}/stateless/deployment-template.yaml[tag=config-block]
+include::{configDir}/stateless/deployment.yaml[]
```

Comment:
There already exists a template file - we should use it instead of adding new files.